ten-simple-rules-dockerfiles.Rmd

```diff
@@ -326,7 +327,7 @@ For example, here is how we would use a base image `verse`, which provides the p
 \footnotesize
 
 ```dockerfile
-FROM docker.io/rocker/r-ver:3.5.2
+FROM docker.io/rocker/verse:3.6.2
```

Comment:
Do we talk anywhere about if there is a standard / best practice for including a registry? My rule of thumb is to leave out docker since the Docker registry is default (so this would start with "rocker") and then only add it for a different registry (quay.io, which I've been using more because it let's me create repo and permission specific tokens). I haven't seen many containers in the wild that explicitly state docker.io, actually, and if you added this to a container from a few years back it would have been something like [registry-2.docker.io](https://github.com/hpcng/singularity/blob/vault/release-2.2.x/libexec/python/docker/api.py#L32). So in that docker has historically changed this (but we could always consider whatever the Docker registry is to be the default) my inclination is to leave the specific registry string out for the longevity of the paper. Thoughts?

Reply:
I concur not many Dockerfiles in the wild do this, but IMO if Docker (again) changes this in the future and Dockerfiles break because of that, they at least should break with a helpful error. The information is also more explicit (seeing a Dockerfile with registry-2.docker.io today tells me something about the file!). So: agree it is not common practice, but it's more explicit.

Reply:
Let's leave this to for the reviewers to give their opinion on.